FLT/Mathlib/Data/ENNReal/Inv.lean

```diff
@@ -0,0 +1,8 @@
+import Mathlib.Data.ENNReal.Inv
+
+namespace ENNReal
+
+protected lemma mul_div_cancel_right {b : ℝ≥0∞} (hb₀ : b ≠ 0) (hb : b ≠ ∞) (a : ℝ≥0∞) :
+    a * b / b = a := by rw [ENNReal.mul_div_right_comm, ENNReal.div_mul_cancel hb₀ hb]
```

Comment:
Can you clarify whether (a) this is already a PR to mathlib and (b) whether you are going to make this PR or are leaving it so someone else? Same comment for every file in `FLT/Mathlib`.

Reply:
My plan is to equip FLT with [the upstreaming dashboard which I have developed for LeanCamCombi](https://yaeldillies.github.io/LeanCamCombi/upstreaming) (warning: not looking at its best right now, Javier and I will fix it later today), meaning that anyone will be able to upstream those files to mathlib asynchronously.

I personally haven't planned yet to PR that to mathlib, but once there are enough things in the Mathlib folder I will open series of PRs with that material.